src/BUILD

```
@@ -397,7 +410,16 @@ cc_library(
         "-Wall",
         "-Wno-unknown-pragmas",
         "-Werror",
-    ],
+        ] + select({
+            "//conditions:default": [
+            ],
+            "//src:fuzzer_build" : [
+                "-fsanitize=address,undefined,",
```

address sanitizer and ub sanitizer can't be enabled at the same time
```suggestion
                "-fsanitize=address",
```